client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/hooks/use-recipe.ts

```diff
@@ -114,7 +114,7 @@ const useRecipe = (
 		setSearchParams(
 			makeSearchParams( ( currentSearchParams ) => {
 				if ( variation ) {
-					currentSearchParams.set( 'style_variation', variation.slug );
+					currentSearchParams.set( 'style_variation', variation.slug ?? '' );
```

Comment:
Not sure if we can set the query param to an empty string.... just to be very sure maybe just change the condition to `if ( variation?.slug )` ?